modules/javafx.base/src/test/java/test/javafx/util/converter/LocalDateTimeStringConverterTest.java

```diff
@@ -155,4 +157,15 @@ public void fromString_testInvalidInput() {
         assertEquals("12 January 1985, 12:34:56", converter.toString(VALID_LDT_WITH_SECONDS));
         assertEquals(VALID_LDT_WITH_SECONDS, converter.fromString("12 January 1985, 12:34:56"));
     }
+
+    @Test
+    public void testChronologyConsistency() {
+        var converter = new LocalDateTimeStringConverter(FormatStyle.FULL, FormatStyle.MEDIUM, null, JapaneseChronology.INSTANCE);
+        assertEquals(JAPANESE_DATE_STRING, converter.toString(VALID_LDT_WITH_SECONDS));
+        // force a chronology change with an invalid Japanese date
+        try {
+            System.out.println(converter.toString(LocalDateTime.of(1, 1, 1, 1, 1, 1)));
```

Comment:
Since you are expecting an exception, it seems a good idea to fail if you didn't get one. Something like:
```
    fail("Did not get the expected exception");
```
Also, you might consider narrowing the `catch` to just the exception that you expect, so that an unexpected exception will also fail the test.

Reply:
The test checks that the chronology didn't change as a result of catching the exception internally. The thrown exception is from `formatter.parse` later on and will happen regardless of this fix. The only reason I catch and ignore an exception here is so that we can reach the assertion line after that.

If we want to test that an exception is thrown, it should be as part of an input-output test (and we can do the same for other converters).